great_expectations/data_context/store/database_store_backend.py

```diff
@@ -31,21 +34,23 @@
 class DatabaseStoreBackend(StoreBackend):
     def __init__(
         self,
-        credentials,
         table_name,
         key_columns,
         fixed_length_key=True,
-        suppress_store_backend_id=False,
-        manually_initialize_store_backend_id: str = "",
+        credentials=None,
+        url=None,
+        connection_string=None,
+        engine=None,
         store_name=None,
+        suppress_store_backend_id=False,
```

Comment:
`store_name` and `supress_store_backend_id` were re-added to the signature due to tests failing.